acceptance/GK_welcome.robot

```diff
@@ -0,0 +1,41 @@
+*** Settings ***
+Library         SeleniumLibrary  timeout=10  implicit_wait=0
+Resource        ./functions/PageWelcome.robot
+Test Teardown   Close Browser
+Force Tags      Welcome
+
+*** Test Cases ***
+
+Welcome: (EN)
+  [Documentation]    Default english welcome page
+  [Tags]             EN
+  !Go To GeoKrety
+  Welcome WaitForPage
+  Welcome ShouldShow Geokrety
+  Welcome ShouldShow News
+  Welcome ShouldShow RecentLogs
+  Welcome ShouldShow RecentImages
+  Welcome ShouldShow RecentGK
+
+Welcome: (FR)
+  [Documentation]    Welcome page in french
+  [Tags]             FR
+  !Go To GeoKrety
+  !Click On FR Flag
+  Welcome WaitForPage
+  Welcome ShouldShow GeokretyFR
+  Welcome ShouldShow NewsFR
+  Welcome ShouldShow RecentLogsFR
+  Welcome ShouldShow RecentGKFR
+
+Welcome: input geokret code
+  [Documentation]    Traking code should redirect to ruchy

```

Comment:
Traking -> Tracking

Reply:
good catch thanks, will rebase squash and amend tomorrow